xbit/common/xbitLogger.js

```diff
@@ -0,0 +1,12 @@
+var config = require('config');
+var bunyan = require('bunyan');
+
+var streams = [{stream: process.stdout}];
+config.logFilePath && streams.push({path: config.logFilePath});
+
+var logger = bunyan.createLogger({
+    name: 'xbitApp',
```

Comment:
The name field is very useful as it will show up in the log
So we shall utilize this field to contains more information like 
process (process id), module name etc. 

Eventually we shall use node cluster to scale the APP